Mathlib/Mathport/Syntax.lean

```diff
@@ -171,7 +171,8 @@ namespace Tactic
 /- E -/ syntax (name := mapply) "mapply " term : tactic
 /- M -/ syntax (name := withCases) "with_cases " tacticSeq : tactic
 syntax caseArg := binderIdent,+ (" :" (ppSpace (ident <|> "_"))+)?
-/- N -/ syntax (name := case') "case' " (("[" caseArg,* "]") <|> caseArg) " => " tacticSeq : tactic
+-- [todo] case' is now present in Lean core
+-- /- N -/ syntax (name := case') "case' " (("[" caseArg,* "]") <|> caseArg) " => " tacticSeq : tactic
```

Comment:
```suggestion
-- [todo] case' is now present in Lean core
-- /- N -/ syntax (name := case') "case' " (("[" caseArg,* "]") <|> caseArg) " => " tacticSeq : tactic
```
```suggestion
/- N -/ syntax (name := case'') "case'' " (("[" caseArg,* "]") <|> caseArg) " => " tacticSeq : tactic
```
I very much doubt the one in lean core has exactly this syntax and behavior.

Reply:
What's the right thing to do in this case'?

Reply:
like my suggestion: just add more primes